### PR TITLE
Feature/CON-30/support responsive video Item Authoring

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/CON-30

- [x] https://github.com/oat-sa/tao-item-runner-qti-fe/pull/140

https://github.com/oat-sa/tao-core/pull/2780
https://github.com/oat-sa/tao-core-ui-fe/pull/243
https://github.com/oat-sa/extension-tao-mediamanager/pull/248
https://github.com/oat-sa/extension-tao-test-preview-ui-loader/pull/17

Support video responsive width in Item Authoring.

How to test:

- go to Items Authoring
- add video by Media Interaction and by media object
- try to scale
- check scale in preview

### CODE REVIEW

File: views/js/qtiCreator/tpl/forms/static/object.tpl [hunk 1]
```diff
@@ -19,4 +19,11 @@
...
                data-max="1080"/>
     </div>
+</div>
+
+<div class="panel media-size-panel">
+    <h3>{{__ 'Size and position'}}</h3>
+    <div class="media-sizer">
+        <!-- mediaEditorComponent goes here -->
+    </div>
 </div>
```
Comment (habemuscode) [inline views/js/qtiCreator/tpl/forms/static/object.tpl:29]: Missing end line

**Review by habemuscode — APPROVED** [review]
Validated on: http://test-hanna.playground.kitchen.it.taocloud.org:40141 (Request access to @bziondik)

- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: views/package.json [hunk 2]
```diff
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "0.6.1",
-        "@oat-sa/tao-item-runner-qti": "0.16.0"
+        "@oat-sa/tao-item-runner-qti": "git://github.com/oat-sa/tao-item-runner-qti-fe#feature/CON-30/responsive-video"
```
Comment (404passionFruit) [inline views/package.json:12]: Shouldn't it be a version number?

**Review by 404passionFruit — COMMENTED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by 404passionFruit — APPROVED** [review]

**Pull request merged**